target/linux/ath79/dts/qca9563_compex_wpj563.dts

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "Compex WPJ563";
+	compatible = "compex,wpj563", "qca,qca9563";
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		sig1 {
+			label = "wpj563:green:sig1";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		sig2 {
+			label = "wpj563:green:sig2";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		sig3 {
+			label = "wpj563:green:sig4";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+		};
+
+		sig4 {
+			label = "wpj563:green:sig4";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+		};
+	};
```

Comment:
What about the buzzer set up in ar71xx?

Reply:
completely forgot about that :speak_no_evil: